examples/benchmark/benchmark.js

```diff
@@ -0,0 +1,62 @@
+/* global Benchmark:false, uuidv1:false, uuidv3:false, uuidv4:false, uuidv5:false */
+const Benchmark = (typeof window !== 'undefined' && window.Benchmark) || require('benchmark');
+const uuidv1 = (typeof window !== 'undefined' && window.uuidv1) || require('uuid').v1;
+const uuidv4 = (typeof window !== 'undefined' && window.uuidv4) || require('uuid').v4;
+const uuidv3 = (typeof window !== 'undefined' && window.uuidv3) || require('uuid').v3;
+const uuidv5 = (typeof window !== 'undefined' && window.uuidv5) || require('uuid').v5;
+
+const MY_NAMESPACE = '55238d15-c926-4598-b49d-cf4e913ba13c';
```

Comment:
No need to performance test DNS, URL, and custom namespaces.  The codepath will be the same regardless of the namespace id, so just pick one (e.g. DNS) and test that.

```suggestion
```